docs/Clearwater_stress_testing.md

```diff
@@ -12,82 +12,78 @@ package installed.
+End: 2016-11-16 15:18:36.414653
 
-### Manual (i.e. non-Chef) stress runs
+Total calls: 1625
+Successful calls: 1612 (99.2%)
+Failed calls: 13 (0.8%)
 
-SIP stress can also be run against a deployment that has been installed manually (as per the [Manual Install instructions](Manual_Install.md)).
+Retransmissions: 0
 
-Firstly follow [this process](https://github.com/Metaswitch/crest/blob/dev/docs/Bulk-Provisioning%20Numbers.md) to bulk provision subscribers. Work out how many stress nodes you want, and create 30000 subscribers per SIPp node.
+Average time from INVITE to 180 Ringing: 74.0 ms
+# of calls with 0-2ms from INVITE to 180 Ringing: 0 (0.0%)
+# of calls with 2-20ms from INVITE to 180 Ringing: 0 (0.0%)
+# of calls with 20-200ms from INVITE to 180 Ringing: 1572 (96.7384615385%)
+# of calls with 200-2000ms from INVITE to 180 Ringing: 45 (2.76923076923%)
+# of calls with 2000+ms from INVITE to 180 Ringing: 0 (0.0%)
 
-To create a new SIPp node, create a new virtual machine and bootstrap it [by configuring access to the Project Clearwater Debian repository](Manual_Install.md#configure-the-apt-software-sources).
+Total re-REGISTERs: 1800
+Successful re-REGISTERs: 1792 (99.5555555556%)
+Failed re-REGISTERS: 8 (0.444444444444%)
 
-Then set the following properties in /etc/clearwater/local_config:
+REGISTER retransmissions: 0
 
-* (required) local_ip - the local IP address of this node
-* (optional) node_idx - the node index (defaults to 1)
+Average time from REGISTER to 200 OK: 15.0 ms
 
-Set the following properties in /etc/clearwater/shared_config:
+Log files in /var/log/clearwater-sip-stress
+```
 
-* (required) home_domain - the home domain of the deployment under test
-* (optional) bono_servers - a list of bono servers in this deployment
-* (optional) stress_target - the target host (defaults to the $node_idx-th entry in $bono_servers or, if there are no $bono_servers, defaults to $home_realm)
-* (optional) base - the base directory number (defaults to 2010000000)
-* (optional) count - the number of subscribers to run on this node (must be even, defaults to 30000)
+### Extra run_stress options
 
-Finally, run `sudo apt-get install clearwater-sip-stress` to install the Debian package. Stress will start automatically after the package is installed.
+The run-stress script has some command-line options:
 
-### Configuring UDP Stress
+* `--initial-reg-rate` - this controls how many REGISTERs/second the script sends in during the initial registration phase (defaulting to 80). On systems that can cope with the load, raising this value will let the test run start faster.
+* `--sipp-output` - By default, the script hides the SIPp outputand just presents the end-of-run stats. With this option, it will show the SIPp output screen, which may be useful for users familiar with SIPp.
+* `--target TARGET` - Domain/IP and port to target stress at. Default is sprout.{domain}, with I-CSCF traffic going to port 5052 and S-CSCF traffic to port 5054.
```

Comment:
How does this work for I-CSCF/S-CSCF, if all you can add is a single domain:port?